gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,9 +1,8 @@
-#Wed May 08 10:57:16 CEST 2019
+#Tue Mar 24 09:13:30 GMT 2020
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
 android.enableD8.desugaring=true
-distributionUrl=https\://services.gradle.org/distributions/gradle-5.6.3-all.zip
-# Android Studio Gradle wrapper >= 3.1.2 doesn't support configure on demand
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.6.4-all.zip
```

Comment:
Why not the latest 6.2.2?

Reply:
Huh that was auto generated. Probably just game me the minimum for that plugin version. I'll update it.

Reply:
Wow this is causing all kinds of problems. Did you try and update spotbugs at all as part of this? It looks like we might be using a very old version.

Reply:
ahh spotbugs... I think I was able to update it to version 2.0.0 but no the latest 4.0.4